packages/palette/src/elements/PhoneInput/PhoneInput.tsx

```diff
@@ -93,28 +106,13 @@ export const PhoneInput = React.forwardRef<HTMLInputElement, PhoneInputProps>(
     })
 
     const { anchorRef, tooltipRef } = usePosition({
-      key: filteredOptions.length,
       position: "bottom",
       offset: 10,
       active: isDropdownVisible,
       flip: false,
     })
 
-    const { reset, set } = useKeyboardListNavigation({
-      ref: containerRef,
-      list: filteredOptions,
-      waitForInteractive: true,
-      onEnter: ({ element: option, event }) => {
-        event.preventDefault()
-        event.stopPropagation()
-        if (option) {
-          handleSelect(option)
-        }
-        resetUI()
-      },
-    })
```

Comment:
Why would we want to remove keyboard nav?

Reply:
Also I wrote this library :) if there's a problem with it, it's fixable. But it's very simple!

Reply:
Yeah, whoever implemented this kind of just missed the point of the lib. Notice how the thing you use to actually highlight things isn't even destructured.